includes/sanitizers/class-amp-allowed-tags-generated.php

```diff
@@ -6756,7 +6777,6 @@ class AMP_Allowed_Tags_Generated {
 							2,
 							3,
 							7,
-							9,
```

Comment:
It seems `amp-video` still doesn't support the intrinsic layout.

Reply:
Ah never mind this spec is for `amp-story >> amp-story-page-attachment >> amp-video`.

Reply:
@westonruter should the intrinsic layout be applied to this spec rule also? It was removed in the [same commit](https://github.com/ampproject/amphtml/commit/d6590408cc04ce0f0bcb34cd8fded2e7fea696dc#diff-26f44ce4399084028b4948a3ed1cae1b96e7d73587c2d600022900e12ca49e17) that removed it for the `amp-video` spec rule.

Reply:
It was originally introduced in https://github.com/ampproject/amphtml/commit/e400a655258d614a6248fcb259aee2a11e70bf9f.

Reply:
Yes, I think you're right. Noted in https://github.com/ampproject/amphtml/pull/31373/files#r553553151.

Reply:
Restored in https://github.com/ampproject/amphtml/pull/31922.